CHANGELOG.asciidoc

```diff
@@ -280,6 +280,10 @@ specifying links during span creation (with the limitation that span link
   Lambda layers now that
   https://aws.amazon.com/blogs/compute/node-js-16-x-runtime-now-available-in-aws-lambda/[this runtime is available on AWS].
 
+- Add `*principal*` pattern to default value for `sanitizeFieldNames` config
```

Comment:
Slightest of nits with the weight of a photon -- but did you want `*` instead of `-` here for the list? (based on previous commit feedback)

Reply:
You are right. Thanks for pointing it out.

(Also I added this bullet in totally the wrong section (it was low down in a section for an earlier release).

Reply:
commit 040cc180